lib/compiler-finder.ts

```diff
@@ -345,6 +345,7 @@ export class CompilerFinder {
                 preamble: props<string>('licensePreamble'),
             },
             possibleOverrides: [],
+            activeOverrides: [],
```

Comment:
This is too confusing of a name, once this is transmitted to the frontend this object should not ever change. But then the Default overrides are also already in `possibleOverrides`, so I'm not sure what's the point of this.

Reply:
Not sure I understand this.
IIUC this (compiler-finder stuff) runs only on instance startup. Why shouldn't `activeOverrides` change afterwards?
And what better name would you suggest?

Reply:
Because it's supposed to represent the static configuration on the backend for everyone, not an individualized state

Reply:
Got it. Will take another stab at this, probably next weekend